src/webrtc/groupCallEventHandler.ts

```diff
@@ -46,7 +47,25 @@ export class GroupCallEventHandler {
 
     constructor(private client: MatrixClient) { }
 
-    public start(): void {
+    public async start(): Promise<void> {
```

Comment:
Does this handle the client sending the sync request but not getting a response and processing it yet?

Reply:
The sync event is emitted after each sync result is processed, so it will wait until the first sync result is fully processed.